### PR TITLE
Allow the config file to get connection information from the .env file

### PR DESCRIPTION
**Pull request by denischatelain**
Enable the use of the .env file to configure the rabbitmq connection

### CODE REVIEW

**Comment by stevenklar** [comment]
@denischatelain Could you also change the README accordingly with the lumen configuration example?

**Comment by bschmitt** [comment]
@denischatelain thanks for your contribution! Do you mind to change the names form `RABBITMQ_`  to `AMQP_` as this package is theoretically usable by every message queue which supports AMQP. Thank you!

File: config/amqp.php [hunk 1]
```diff
@@ -19,11 +19,11 @@
     'properties' => [
 
         'production' => [
-            'host'                  => 'localhost',
-            'port'                  => 5672,
-            'username'              => '',
-            'password'              => '',
-            'vhost'                 => '/',
+            'host'                  => env('RABBITMQ_HOST', 'localhost'),
```
Comment (stevenklar) [inline config/amqp.php]: Can you please change the env prefix from ```RABBITMQ_``` to ```AMQP_```

Comment (stevenklar) [inline config/amqp.php]:
```suggestion
            'host'                  => env('AMQP_HOST', 'localhost'),
```

File: config/amqp.php [hunk 2]
```diff
@@ -19,11 +19,11 @@
...
+            'port'                  => env('RABBITMQ_PORT', 5672),
```
Comment (stevenklar) [inline config/amqp.php]:
```suggestion
            'port'                  => env('AMQP_PORT', 5672),
```

File: config/amqp.php [hunk 3]
```diff
@@ -19,11 +19,11 @@
...
+            'username'              => env('RABBITMQ_LOGIN', 'guest'),
```
Comment (stevenklar) [inline config/amqp.php]:
```suggestion
            'username'              => env('AMQP_LOGIN', 'guest'),
```

File: config/amqp.php [hunk 4]
```diff
@@ -19,11 +19,11 @@
...
+            'password'              => env('RABBITMQ_PASSWORD', 'guest'),
```
Comment (stevenklar) [inline config/amqp.php]:
```suggestion
            'password'              => env('AMQP_PASSWORD', 'guest'),
```

File: config/amqp.php [hunk 5]
```diff
@@ -19,11 +19,11 @@
...
+            'vhost'                 => env('RABBITMQ_VHOST', '/'),
```
Comment (stevenklar) [inline config/amqp.php]:
```suggestion
            'vhost'                 => env('AMQP_VHOST', '/'),
```

**Comment by stevenklar** [comment]
@denischatelain can you review the suggestions so we can merge your contribution, please?

**Comment by andrew-svirin** [comment]
What will happen if  in config will setup: `'use' => 'local' `

**Comment by stevenklar** [comment]
Closed in favour of PR #86

**Pull request closed without merging**